PencilCase/PencilCase/scripts/Pages/DealPageBase.ts

```diff
@@ -92,34 +133,39 @@ export class DealPageBase extends PageBase {
     }
 
     protected increaseQuantity = () => {
-        this.selectedProductQuantity(this.selectedProductQuantity() + 1);
+        this.selectedProductQuantity(Number(this.selectedProductQuantity()) + 1);
```

Comment:
Seems if you type the value in the text box, and then it will handled the value as string. For example, you typed 20, and then click the increase button to expected it to be 21, but it will be 201. Has no idea why this happens. 
To fix this issue, always add this Number(string) converter to explicitly convert the value as number.